TEAM.md

```diff
@@ -15,6 +15,6 @@ Starbeam is developed and maintained by the Starbeam team.
     <td>
       <img src="https://github.com/starbeamjs/.github/assets/4/ec5e50ee-c83d-4efd-abf1-aa85a0d5d1eb" alt="nullvoxpopuli" width="100">      
     </td>
-    <td>Yehuda Katz (<a href="https://github.com/nullvoxpopuli">@nullvoxpopuli</a>)</td>
+    <td>Preston Sego (<a href="https://github.com/nullvoxpopuli">@nullvoxpopuli</a>)</td>
```

Comment:
```suggestion
    <td>NullVoxPopuli (<a href="https://github.com/nullvoxpopuli">@nullvoxpopuli</a>)</td>
```

Reply:
my real name isn't all that well known (defo not secret, tho), so for SEO reasons, let's stick with NVP <3

Thanks for updating!!!